src/artemis/experiment_plans/rotation_scan_plan.py

```diff
@@ -70,27 +75,63 @@ def cleanup_sample_environment(
     yield from bps.abs_set(detector_motion.shutter, 0, group=group)
 
 
+def move_x_y_z(
+    smargon: Smargon,
+    x: float | None,
+    y: float | None,
+    z: float | None,
+    wait=False,
+    group="move_x_y_z",
+):
+    if x:
+        LOGGER.info(f"x: {x}, y: {y}, z: {z}")
+        yield from bps.abs_set(smargon.x, x, group=group)
+    if y:
+        yield from bps.abs_set(smargon.y, y, group=group)
+    if z:
+        yield from bps.abs_set(smargon.z, z, group=group)
+    if wait:
+        yield from bps.wait(group)
```

Comment:
Should: It pains me how much this is like `move_xyz` I don't want to end up writing a new one for every plan. Can we either combine them or put them in some utils location where the names are more obviously distinct about what they do. I also think separate unit tests on this function would be good too.

Reply:
Covered by https://github.com/DiamondLightSource/python-artemis/issues/834